walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/Event.java

```diff
@@ -24,6 +25,14 @@
  * busy. Events are considered read-only.
  */
 public final class Event {
+
+  public static final Comparator<Event> EVENT_START_COMPARATOR = new Comparator<Event>() {
+    @Override
+    public int compare(Event a, Event b) {
+      return TimeRange.ORDER_BY_START.compare(a.getWhen(), b.getWhen());
+    }
+  };
```

Comment:
Turns out there's an easier way, just use:

```
Comparator.comparing(Event::getWhen, TimeRange.ORDER_BY_START)
```

See https://docs.oracle.com/javase/8/docs/api/java/util/Comparator.html#comparing-java.util.function.Function-java.util.Comparator-

Reply:
Yes You can use Java 8 comparators using lambda expressions.